### PR TITLE
[Edit] CSS display flex example image addition

### PR DESCRIPTION
**Pull request by Khalid-debugg**
### Description

Edited display.md
closes issue #3740 

### Type of Change

- Editing an existing entry (fixing a typo, bug, issues, etc)

### Checklist

- [x] All writings are my own.
- [x] My entry follows the Codecademy Docs style guide.
- [x] My changes generate no new warnings.
- [x] I have performed a self-review of my own writing and code.
- [x] I have checked my entry and corrected any misspellings.
- [x] I have made corresponding changes to the documentation if needed.
- [x] I have confirmed my changes are not being pushed from my forked `main` branch.
- [x] I have confirmed that I'm pushing from a new branch named after the changes I'm making.
- [x] Under "Development" on the right, I have linked any issues that are relevant to this PR (write "Closes #<issue number> in the "Description" above).

### CODE REVIEW

File: content/css/concepts/display/display.md [hunk 1]
```diff
@@ -214,3 +214,5 @@ Setting a `div` element to behave like a flex box.
   display: flex;
 }
 ```
+
```
Comment (ishg-153) [inline content/css/concepts/display/display.md:217]: Add in a sentence that leads the above code block to this output. Something like this:
"This does [XYZ] and results in:"

File: content/css/concepts/display/display.md [hunk 2]
```diff
@@ -214,3 +214,5 @@ Setting a `div` element to behave like a flex box.
...
+![css-display-flex-example](https://raw.githubusercontent.com/Codecademy/docs/main/media/css-display-flex-example.jpg)
```
Comment (ishg-153) [inline content/css/concepts/display/display.md:218]:
```suggestion
![CSS display flex example](https://raw.githubusercontent.com/Codecademy/docs/main/media/css-display-flex-example.jpg)
```
Inside the square brackets is the alt text for the image. It should be a short phrase relating to the image, like so:

**Review by ishg-153 — CHANGES_REQUESTED** [review]
@Khalid-debugg I have suggested a few changes. Please make those.
Also, make sure to sign the CLA.

**Review by caupolicandiaz — APPROVED** [review]
@Khalid-debugg For the sake of expediency I updated the image for a higher resolution copy. Looks good to merge

**Comment by ishg-153** [comment]
> @Khalid-debugg @ishg-153 For the sake of expediency I updated the image for a higher resolution copy. Looks good to merge

Hey, there seems to be a small grammatical error in the sentence above the image, can we correct that?

**Comment by caupolicandiaz** [comment]
> > @Khalid-debugg @ishg-153 For the sake of expediency I updated the image for a higher resolution copy. Looks good to merge
> 
> Hey, there seems to be a small grammatical error in the sentence above the image, can we correct that?

Feel free to add it.

**Comment by ishg-153** [comment]
> > > @Khalid-debugg @ishg-153 For the sake of expediency I updated the image for a higher resolution copy. Looks good to merge
> > 
> > 
> > Hey, there seems to be a small grammatical error in the sentence above the image, can we correct that?
> 
> Feel free to add it.

@Khalid-debugg made the necessary change, it's ready to merge.

**Comment by Khalid-debugg** [comment]
> > > > @Khalid-debugg @ishg-153 For the sake of expediency I updated the image for a higher resolution copy. Looks good to merge
> > > 
> > > 
> > > Hey, there seems to be a small grammatical error in the sentence above the image, can we correct that?
> > 
> > 
> > Feel free to add it.
> 
> @Khalid-debugg made the necessary change, it's ready to merge.

sorry but I can't see a grammatical mistake, I even used an external software and I can't figure it out, can you point it out?

**Review by ishg-153 — APPROVED** [review]
Good to go

**Pull request merged**